src/Collector/ClassNameCollector.php

```diff
@@ -9,22 +9,11 @@
 
 class ClassNameCollector implements CollectorInterface
 {
-
-
     public function getType()
     {
         return 'className';
     }
 
-    private function getRegexByConfiguration(array $configuration)
```

Comment:
half useless here but well: removed extra spaces and moved the private function *after* the public one (public is more important than private no?). Well personal opinion maybe I shouldn't propose such changes...